sources/ade/source/execution_engine.cpp

```diff
@@ -138,7 +138,7 @@ std::unique_ptr<Executable> ExecutionEngine::createExecutable(const Graph& graph
         }
     }
 
-    return ret;
+    return std::move(ret);
```

Comment:
This definitely won't do like that. Let's add at least a comment why this move is required, otherwise a new commit like https://github.com/opencv/ade/commit/e80509b09fe350b60fe85c5e7849cb8a9c65c93d will appear and we'll get the same problem again

Reply:
by the way, I wonder if we can ensure there's no "redundant-move" warning appears? may be we can patch it somehow so that this thing is absolutely necessary, e.g. would the following work?:
```cpp
return std::unique_ptr<...>(std::move(ret));  // the new object must be an rvalue, right?!
```